common/placer1.cc

```diff
@@ -225,7 +225,8 @@ class SAPlacer
             // Once cooled below legalise threshold, run legalisation and start requiring
             // legal moves only
             if (temp < legalise_temp && !require_legal) {
-                legalise_design(ctx);
+                legalise_relative_constraints(ctx);
+                // FIXME
```

Comment:
What does this refer to?

Reply:
Obsolete, from before I put the legalise step in there again